snooty.toml

```diff
@@ -63,8 +63,10 @@ backend-short = "App Services"
 base-url = "https://www.mongodb.com/docs/atlas/app-services"
 charts = "MongoDB Charts"
 charts-short = "Charts"
-cli = "Realm CLI"
-cli-bin = ":binary:`realm-cli`" # binary -- DO NOT USE IN LINKS! Will break them.
+cli = "App Services CLI"
+cli-bin = "app-services-cli"
+cli-bin-mono = "``app-services-cli``"
```

Comment:
stylistically i think this should pretty much never be used. i understand that you're substituting existing similar text here (i.e `realm-cli`), but i think most all of these instances should be replaced by just the "App Services CLI".